packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -97,25 +99,10 @@ export function PanelChrome({
   };
 
   const containerStyles: CSSProperties = { width, height };
-
-  const isUsingDeprecatedLeftItems = isEmpty(status) && !loadingState;
-  const showLoading = loadingState === LoadingState.Loading && !isUsingDeprecatedLeftItems;
-  const showStreaming = loadingState === LoadingState.Streaming && !isUsingDeprecatedLeftItems;
```

Comment:
Since loadingState is a new prop I don't think we need this extra "!isUsingDeprecatedLeftItems" check as well, if you pass loadingState we use it!

Reply:
yes, absolutely, thank you for taking care of it!